db/schema.rb

```diff
@@ -1242,8 +1250,10 @@
   add_foreign_key "oauth_access_grants", "users", column: "resource_owner_id"
   add_foreign_key "oauth_access_tokens", "oauth_applications", column: "application_id"
   add_foreign_key "oauth_access_tokens", "users", column: "resource_owner_id"
+  add_foreign_key "optional_fields", "users"
   add_foreign_key "page_views", "articles", on_delete: :cascade
   add_foreign_key "podcasts", "users", column: "creator_id"
+  add_foreign_key "pro_memberships", "users"
```

Comment:
this probably snuck in from some previous migration

Reply:
Whoops, yeah, not sure how that got there, I'll take it out.